mkosi.finalize

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+[ "$1" == "final" ] || exit 0
+
+exit 0
```

Comment:
Everything after this `exit 0` is ignored. Please either remove the line or the file altogether.

Reply:
I've removed the file